soroban-sdk/src/storage.rs

```diff
@@ -148,9 +148,9 @@ impl Storage {
     /// the function including the current ledger. This means the last ledger
     /// that the entry will be accessible will be the current ledger sequence
     /// plus the max expiration minus one.
-    pub fn max_expiration(&self) -> u32 {
+    pub fn max_live_until_ledger(&self) -> u32 {
```

Comment:
This should be `max_ttl`. We're not returning a ledgerSeq, which would be a `live_until_ledger`. We are returning an amount of ledgers, which is referred to as TTL. We may want to change the comment to also discuss the TTL. I expect developers to interact primarily in terms of TTL, not raw `live_until_ledger` sequence numbers.